IOSExercises/Classes/LocaleManager.m

```diff
@@ -0,0 +1,50 @@
+//
+//  LocaleManager.m
+//  IOSExercises
+//
+//  Created by Arnold Joseph Caesar Esteban on 11/4/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "LocaleManager.h"
+
+@interface LocaleManager ()
+
+@end
+
+@implementation LocaleManager
+
+static LocaleManager *localeManager = nil;
+
+- (id)init {
```

Comment:
Singleton pattern is usually done in factory method, not in init of an instance

For reference, init method usually looks like below (and is only overridden if there is need for additional setup for the instance)
```
- (instancetype)init {
  if (self = [super init]) {
    // Setup for properties used inside the specific instance 
  }
}
```